docroot/modules/humsci/hs_basic_page/hs_basic_page.install

```diff
@@ -18,5 +18,170 @@ function hs_basic_page_install() {
     \Drupal::messenger()
       ->addError(t('Unable to set initial display settings for Basic Page'));
   }
+}
+
+/**
+ * Update field config with paragraphs_browser widget for basic page.
+ */
+function hs_basic_page_update_9001() {
+  $themes_to_update = [
+    'humsci_colorful',
+    'humsci_traditional',
+  ];
+  $modules = [
+    'file',
+    'insert',
+    'paragraphs_browser',
+    'path',
+    'paragraphs',
+  ];
+  // Install only on the specified themes.
+  $site_theme = \Drupal::theme()->getActiveTheme()->getName();
+  if (!in_array($site_theme, $themes_to_update, TRUE)) {
```

Comment:
No need use strict here. it will always be strings.

Reply:
Fixed.